config/config.nims

```diff
@@ -1,5 +0,0 @@
-# this config.nims also needs to exist to prevent future regressions, see #9990
-
-when defined(nimHasCppDefine):
-  cppDefine "errno"
```

Comment:
... because `cppDefine ` is an ugly hack and should be replaced by param mangling or listing of what defines are used by calling `gcc -dM -E` (and only mangling those) but util then isn't that reviving https://github.com/nim-lang/Nim/issues/11153 ?

as for removing config.nims, IIRC the main cost is that nimscript now imports os, but once https://github.com/nim-lang/Nim/issues/14179#issuecomment-625200718 is fixed, that cost goes away (and possibly, nim.cfg can be merged into config.nims if it's negligible zero cost)

Reply:
I renamed the file for my testing purposes and forgot about it, will undo this change.